mocha-multi.js

```diff
@@ -49,6 +53,35 @@ function MochaMulti(runner, options) {
   }
 }
 
+util.inherits(MochaMulti, Base);
```

Comment:
Why is this necessary? I can't see any places where you use a method on the parent